data/json/items/generic.json

```diff
@@ -311,18 +311,22 @@
     "flags": [ "TRADER_AVOID", "FIREWOOD", "TINDER" ]
   },
   {
-    "type": "GENERIC",
+    "type": "AMMO",
     "id": "superglue",
     "category": "spare_parts",
     "symbol": ",",
+    "phase": "liquid",
     "color": "white",
     "name": { "str_sp": "superglue" },
     "description": "A tube of strong glue.  Used in many crafting recipes.",
     "price": "18 USD",
     "price_postapoc": "1 USD",
     "material": [ "plastic" ],
```

Comment:
You probably want to make it no longer plastic if you're making it the liquid and not the entire bottle

Reply:
I'm unsure of an appropriate material. Would water be ok?

Reply:
> I'm unsure of an appropriate material. Would water be ok?

Probably not at all, honestly. Now that I think about it, plastic might be the closest thing we have in terms of materials...